src/widgets/conditions/conditions-form.component.tsx

```diff
@@ -0,0 +1,244 @@
+  const [patientCondition, setPatientCondition] = React.useState(null);
+  const [conditionUuid, setConditionUuid] = React.useState(null);
+  const [conditionClinicalStatus, setConditionClinicalStatus] = React.useState(
+    null
+  );
+  const [viewEditForm, setViewEditForm] = React.useState(true);
+  const [enableEditButtons, setEnableEditButtons] = React.useState(true);
+  const [patientUuid] = useCurrentPatient();
+  const formRef = React.useRef<HTMLFormElement>(null);
+
+  React.useEffect(() => {
+    const abortController = new AbortController();
+    if (patientUuid && props.match.params) {
+      getConditionByUuid(
+        props.match.params["conditionUuid"],
+        abortController
+      ).then(condition => setPatientCondition(condition), createErrorHandler);
+    }
+  }),
+    [patientUuid, props.match.params];
+
+  React.useEffect(() => {
+    const params: any = props.match.params;
+    if (params.conditionUuid) {
+      setViewEditForm(true);
+    } else {
+      setViewEditForm(false);
+    }
+  }, [props.match.params]);
+
+  React.useEffect(() => {
+    if (patientCondition) {
+      setConditionUuid(patientCondition.resource.id);
+      setConditionClinicalStatus(patientCondition.resource.clinicalStatus);
+    }
+  }, [patientCondition]);
+
+  React.useEffect(() => {
+    if (conditionUuid) {
+      setEnableEditButtons(false);
+    } else {
+      setEnableEditButtons(true);
+    }
+  }, [conditionUuid]);
+
+  function createCondition() {
+    return (
+      <SummaryCard name="Add Condition" match={props.match}>
+        <h4>Condition</h4>
+        <p>Add Condition</p>
+      </SummaryCard>
+    );
+  }
+
+  const handleCancelChanges = () => {
+    formRef.current.reset();
+    props.entryCancelled();
+  };
+
+  function editCondition() {
+    return (
+      <SummaryCard
+        name="Edit conditions"
+        match={props.match}
+        styles={{
+          width: "100%",
+          background: "var(--omrs-color-bg-medium-contrast)"
+        }}
+      >
+        {patientCondition && (
+          <form ref={formRef}>
+            <div>
+              <div
+                className={`${styles.conditionEditHeader} omrs-padding-bottom-28`}
+              >
+                <div
+                  className="omrs-type-body-regular"
+                  style={{ color: "#535355" }}
+                >
+                  Condition
+                </div>
+                <div
+                  className={`omrs-type-body-large`}
+                  style={{
+                    color: "var(--omrs-color-ink-high-contrast)",
+                    fontWeight: 500
+                  }}
+                >
+                  {patientCondition.resource.code.text}{" "}
+                </div>
+              </div>
+              <div className={`${styles.conditionHeader} omrs-bold`}>
+                <div
+                  className="omrs-type-body-regular"
+                  style={{ color: "#535355" }}
```

Comment:
use colors defined in the styleguide.

Reply:
This particular color isn't included in the style guide I'm afraid.

Reply:
I've removed this